tests/integration/nmstatectl_test.py

```diff
@@ -282,3 +287,87 @@ def assert_command(cmd, expected_rc=cmdlib.RC_SUCCESS):
 
     assert returncode == expected_rc, cmdlib.format_exec_cmd_result(ret)
     return ret
+
+
+@pytest.fixture
+def eth1_with_static_route_and_rule(eth1_up):
```

Comment:
What about making this tier1 test? It should be really stable

Reply:
Done.